server/create_test.go

```diff
@@ -0,0 +1,108 @@
+package server
+
+import (
+	"bytes"
+	"cmp"
+	"encoding/binary"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/ollama/ollama/api"
+)
+
+func TestConvertFromSafetensors(t *testing.T) {
+	// Store test data in the temp directory
+	t.Setenv("OLLAMA_MODELS", cmp.Or(os.Getenv("OLLAMA_MODELS"), t.TempDir()))
```

Comment:
You don't need to call `cmp.or(os.Getenv("OLLAMA_MODELS), t.TempDir())` here. The reason it's being done in other places is to default to a temporary directory in a test helper in case it's not already set

Reply:
Thanks, made the fix